packages/core/src/components/GridLayout/GridLayout.css

```diff
@@ -0,0 +1,15 @@
+.container {
+  display: grid;
+  height: auto;
+  justify-items: center;
+  align-items:center;
+}
+
+.item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding: 1rem;
```

Comment:
Can we remove this padding and use props to define it?

Reply:
I think we should keep this component just concerned with the layout of the grid, not specific css rules on the grid items, if we allow the parent to pass in children, it can control the padding etc.